### ISSUE TITLE
progressbar==2.3dev is not on PyPI

### ISSUE THREAD

**Issue by antonagestam** [futurecolors/django-geoip#39]
``` shell
Downloading/unpacking progressbar==2.3dev (from django-geoip==0.3.1->-r requirements.txt (line 76))
  Could not find a version that satisfies the requirement progressbar==2.3dev (from django-geoip==0.3.1->-r requirements.txt (line 76)) (from versions: 2.1, 2.2)
  Some externally hosted files were ignored (use --allow-external to allow).
Cleaning up...
No distributions matching the version for progressbar==2.3dev (from django-geoip==0.3.1->-r requirements.txt (line 76))
Storing debug log for failure in /Users/antonagestam/.pip/pip.log
```

**Comment by antonagestam** [comment]
It seems they've rearranged their unstable versions to it's own PyPI repo, I think the new equivalent would be `progressbar-latest==2.3`.

**Comment by coagulant** [comment]
If you're using latest pip maybe, `pip install django-geoip --process-dependency-links`?

**Comment by antonagestam** [comment]
@coagulant I worked around it by installing geoip with --no-deps and manually installing the requirements.

**Comment by coagulant** [comment]
I'll keep this issue until I remove dependency link by vendorizing dependency.
Thank you for bringing the problem up.

**Comment by antonagestam** [comment]
Sounds good, glad to be able to help :)

**Comment by pylemon** [comment]
Same problem here, using `pip install django-geoip --process-dependency-links` works around this problem.

**Comment by coagulant** [comment]
I've just released 0.4 with a fix
https://pypi.python.org/pypi/django-geoip/0.4
